cli/azd/pkg/input/asker.go

```diff
@@ -135,6 +145,33 @@ func askOnePrompt(p survey.Prompt, response interface{}, isTerminal bool, stdout
 			result = v.Default
 		}
 		*pResponse = result
+		return nil
+	case *survey.MultiSelect:
+		// For multi-selection, azd will do a Select for each item, using the default to control the Y or N
+		defValue, err := v.Default.([]string)
+		if !err {
+			return fmt.Errorf("default response type is not a string list '%s'", v.Message)
+		}
+		fmt.Fprintf(stdout, "%s:", v.Message)
+		selection := make([]string, 0, len(v.Options))
+		for _, item := range v.Options {
+			response := slices.Contains(defValue, item)
+			err := askOnePrompt(&survey.Confirm{
```

Comment:
FWIW, I wonder if the programmatic interface would be nicer as:

`My option 1\tMy option 2\n` to mean "select My option 1, My option 2". I choose `\t` as the separator as it's very unlikely to be used in the selection prompt.

The index-ordering based 'y\n' logic is slightly harder to write in my opinion.

Reply:
I don't fully understand.
As a user, I don't need to worry about using a separator for the multiselect.  Azd will do a `Confirm promp` for each of the available options, like:

![image](https://github.com/Azure/azure-dev/assets/24213737/22670fd8-0422-424c-a116-5313f74df4ff)

Each prompt will use the Default list to set either (Y/n or y/N).  So, if I just keep pressing Enter for each prompt, I would get the default selections.
And after each Confirmation, the letter Y or N is displayed below the prompt.

I felt like this was easier from a user point of view than asking for one unique input that contains the items to select.